docs/deploy-worker-udf-binaries.md

```diff
@@ -0,0 +1,101 @@
+  </tr>
+</table>
+
+### 2. Parameter Options
+Once the spark application is bundled, you can launch it using `spark-submit`. The following table shows some of the commonly used options. 
+
+<table>
+  <tr>
+    <td width="25%"><b>Parameter Name</b></td>
+    <td width="75%"><b>Description</b></td>
+  </tr>
+  <tr>
+    <td><b>--class</b></td>
+      <td>The entry point for your application <i>e.g. org.apache.spark.deploy.dotnet.DotnetRunner</i></td>
+  </tr>
+  <tr>
+    <td><b>--master</b></td>
+    <td>The <a href="https://spark.apache.org/docs/latest/submitting-applications.html#master-urls">master URL</a> for the cluster <i>e.g. yarn</i></td>
+  </tr>
+  <tr>
+    <td><b>--deploy-mode</b></td>
+    <td>Whether to deploy your driver on the worker nodes (<code>cluster</code>) or locally as an external client (<code>client</code>). Default: <code>client</code></td>
+  </tr>
+  <tr>
+    <td><b>--conf</b></td>
+      <td>Arbitrary Spark configuration property in <code>key=value</code> format.</br><i>e.g. spark.yarn.appMasterEnv.DOTNET_WORKER_DIR=.\worker\Microsoft.Spark.Worker-0.6.0</i></td>
+  </tr>
+  <tr>
+    <td><b>--files</b></td>
+    <td>Comma-separated list of files to be placed in the working directory of each executor.</br>
+      <li>Please note that this option is only applicable for yarn mode.</li>
+      <li>It supports specifying file names with # similar to Hadoop.</br>
+      <i>e.g. <code>myLocalSparkApp.dll#appSeen.dll</code>. Your application should use the name as <code>appSeen.dll</code> to reference <code>myLocalSparkApp.dll</code> when running on YARN.</i></li></td>
+  </tr>
+  <tr>
+    <td><b>--archives</b></td>
+    <td>Comma separated list of archives to be extracted into the working directory of each executor.</br>
+      <li>Please note that this option is only applicable for yarn mode.</li>
+      <li>It supports specifying file names with # similar to Hadoop.</br>
+      <i>e.g. <code>hdfs://&lt;path to your worker file&gt;/Microsoft.Spark.Worker.net461.win-x64-0.6.0.zip#worker</code>. This will copy and extract the zip file to <code>worker</code> folder.</i></li></td>
+  </tr>
+  <tr>
+    <td><b>application-jar</b></td>
+    <td>Path to a bundled jar including your application and all dependencies.</br>
+    <i>e.g. hdfs://&lt;path to your jar&gt;/microsoft-spark-2.4.x-0.6.0.jar</i></td>
+  </tr>
+  <tr>
+    <td><b>application-arguments</b></td>
+    <td>Arguments passed to the main method of your main class if any</td>
+  </tr>
+</table>
+
+> Note: Please specify all the `--options` before `application-jar` when launching applications with `spark-submit` otherwise they will be ignored. Please see more `spark-submit` options [here](https://spark.apache.org/docs/latest/submitting-applications.html) and running spark on YARN details [here](https://spark.apache.org/docs/latest/running-on-yarn.html).
```

Comment:
```suggestion
> Note: Please specify all the `--options` before `application-jar` when launching applications with `spark-submit`, otherwise they will be ignored. Please see more `spark-submit` options [here](https://spark.apache.org/docs/latest/submitting-applications.html) and running spark on YARN details [here](https://spark.apache.org/docs/latest/running-on-yarn.html).
```

Reply:
Added